manifests/repo/yum.pp

```diff
@@ -5,7 +5,6 @@
 
   if($::mongodb::repo::ensure == 'present' or $::mongodb::repo::ensure == true) {
     yumrepo { 'mongodb':
-      descr          => $::mongodb::repo::description,
```

Comment:
Why should this be removed?

Reply:
Puppet wouldn't deploy the code with it since the mongodb::repo::description didn't exist. Spec tests would pass but on deploy it would complain about accessing a parameter that didn't exist. 

```Error: Could not retrieve catalog from remote server: Error 500 on SERVER: Server Error: Evaluation Error: Unknown variable: '::mongodb::repo::description'. at /etc/puppetlabs/code/environments/database/modules/mongodb/manifests/repo/yum.pp:8:25 on node mongo-db.mydomain.com```

Reply:
Are you perhaps using the latest release since that include 780da69791cfdcb09935df3b2cdbe1619b6e0f3f and fixes the issue properly.

Reply:
No. I forked the codebase back a bit over a month ago to work on getting the Auth settings to work right (see my other pull requests). This was one of several independent fixes I had to make to get to the end goal. I tried to keep them as separate pull requests so that the auth branch work didn't get distracted with several different bug fixes. 

I've drifted a fair amount and several branches away. I could try a resync with the master. If this is fixed elsewhere, happy to let it drop. Real concern is getting the auth branch pulled into master so that we can run MongoDB with auth turned on.

Reply:
I should add that there isn't a pull request for the Auth flag yet because I was looking for additional testing.  I've talked about it and given a reference to my own fork in the nominated bug report (MODULES-5560).  If you're interested in having a more formal look at it, I'll put in a pull request for you.

Reply:
If you could rebase this test and verify it's still needed that would be appreciated.